Python/Product/Common/Parsing/PythonLanguageVersion.cs

```diff
@@ -59,8 +59,9 @@ public static bool IsImplicitNamespacePackagesSupported(this PythonLanguageVersi
             if (Enum.IsDefined(typeof(PythonLanguageVersion), value)) {
                 return (PythonLanguageVersion)value;
             }
-            throw new InvalidOperationException("Unsupported Python version: {0}".FormatInvariant(version));
+            else {
+                return PythonLanguageVersion.None;
```

Comment:
Might be worth adding a `Trace.WriteLine` or something like that here.

Reply:
What's the difference between `Debug.WriteLine` and `Trace.WriteLine`? I know the debug one only runs when debugging, but does Trace write to the VS output window?

Reply:
The only difference is that `Debug` is `[Conditional("DEBUG")]`, and `Trace` is not. Both write to whatever registered trace listeners there are. Under VS, both would go to the Output window.